docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -13,41 +13,39 @@ ms.assetid: 8db75596-19a9-4eda-b20d-a12d517c8cc1
 
 # Getting Started with F# in Visual Studio
 
-F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://visualstudio.com/downloads/).  The rest of the article will assume Visual Studio 2015 Community Edition.
+F# and the Visual F# tooling are supported in the Visual Studio IDE.  To begin, you should [download Visual Studio](https://visualstudio.com/downloads/), if you haven't already.  This article uses the Visual Studio 2015 Community Edition, but you can use F# with the version of your choice.
 
 ## Installing the Visual F# Tools
 
-After Visual Studio is finished initializing, select **Custom** as shown here:
+Visual Studio will first initialize the installer.  After it is intilized, select **Custom** as shown here:
 
-![](../../media/vs2015-install-1.png)
+![](../../media/getting-started-vs/vs2015-install-1.png)
```

Comment:
they need to be in a subfolder where the topic is. so that's why I said the path needs to be:
./media/getting-started-vs/vs2015-install-1.png

I've just followed the guidance for all teams are doing in docs so the experience is consistent.